tests/subsys/llext/simple/src/test_llext_simple.c

```diff
@@ -24,6 +24,20 @@ LOG_MODULE_REGISTER(test_llext_simple);
 #define LLEXT_CONST const
 #endif
 
+#ifdef CONFIG_64BIT
+#define printk_SLID ((const char *)0x87B3105268827052ull)
+#define z_impl_ext_syscall_fail_SLID ((const char *)0xD58BC0E7C64CD965ull)
+#else
+#define printk_SLID ((const char *)0x87B31052ull)
+#define z_impl_ext_syscall_fail_SLID ((const char *)0xD58BC0E7ull)
+#endif
+
+#ifdef CONFIG_LLEXT_EXPORT_BUILTINS_BY_SLID
```

Comment:
Indeed. Fixed.

(I did think about automatically generating these at build-time, but it would cause issues on fresh builds, because you would need the definitions to be able to build to be able to generate the definitions...)